core-bundle/contao/dca/tl_form_field.php

```diff
@@ -226,7 +226,7 @@
 		'customRgxp' => array
 		(
 			'inputType'               => 'text',
-			'eval'                    => array('decodeEntities'=>true, 'maxlength'=>255, 'tl_class'=>'w50', 'mandatory'=>true),
+			'eval'                    => array('useRawRequestData'=>true, 'maxlength'=>255, 'tl_class'=>'w50', 'mandatory'=>true),
```

Comment:
```suggestion
			'eval'                    => array('maxlength'=>255, 'tl_class'=>'w50', 'mandatory'=>true),
```

This should not be necessary, now that we use `StringUtil::decodeEntities()`.

Also you need to base this against `4.13`.

Reply:
I am using this regex `/^[\w .,<>]+$/u` for Contao form field custom validation, allowing only regular alphabats plus . , < > characters.

without `'decodeEntities'=>true` above regex is saved as `/^[&#92;w .,&lt;&#62;]+$/u` in DB
with `'decodeEntities'=>true` it is saved as `/^[\w .,&lt;>]+$/u`


If a user fills form field in frontend with a valid value say `< >` then it is converted/encoded into `&lt; >` by Contao, (but to `&lt; &#62;`, if the form is added to Isotope Checkout, anyway forget this now), now if we use `StringUtil::decodeEntities()`, then we are matching `< >` against above stored value from DB which will fail. Hence my suggestion to add `'useRawRequestData'=>true`.